### PR TITLE
Fix Race Condition for Outdated Drafts

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#19622

*Description of changes:*
- Update dialog action for Corrections race condition

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <BaseDialog
       :setOptions="outOfDateOwnersDialogOptions(getMhrInformation.mhrNumber)"
       :setDisplay="showOutOfDateDraftDialog"
-      @proceed="handleOutOfDateDialogResp"
+      @proceed="handleOutOfDateDialogResp($event)"
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue:21]: The event is already inferred. I'm not sure this changes anything

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1897-adr2y9l4.web.app

File: ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue:21]: Ok, let me try again without this change.

Comment (dimak1) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue:21]: Yes, it works, thanks for reminder, as I was wondering why no event. Closing this PR.

Comment (cameron-eyds) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue:21]: You bet, no prob at all. A nuance of vue templating!

**Comment by dimak1** [comment]
No changes required for Race Condition. Closing this PR.
